contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -344,7 +344,7 @@ function Start-OpenSSHPackage
 
     $buildDir = Join-Path $repositoryRoot ("bin\" + $folderName + "\" + $Configuration)
     $payload =  "sshd.exe", "ssh.exe", "ssh-agent.exe", "ssh-add.exe", "sftp.exe"
-    $payload += "sftp-server.exe", "scp.exe", "ssh-shellhost.exe", "ssh-keygen.exe", "ssh-keyscan.exe" 
```

Comment:
ssh-sk-helper doesn't have the build dependencies defined. This leads to the build errors.
Please move the ssh-sk-helper into the "core" folder.

2>LINK : fatal error LNK1181: cannot open input file 'libssh.lib'
2>Done building project "ssh-sk-helper.vcxproj" -- FAILED.

![image](https://user-images.githubusercontent.com/23668037/145895089-4c18199c-e890-4813-b57a-b1c70b7ff0ad.png)

Reply:
Addressed in ebc802a3. I hadn't tried (at all) to build from Visual Studio, and am a bit surprised that's the only change required. Thanks for pointing it out.

Reply:
I still see ssh-sk-helper ouside of "core" where rest of the project files are defined. Please make this change.

Reply:
can you move the ssh-sk-helper project under "core" similar to other projects like ssh, sshd, etc.

<img width="947" alt="image" src="https://user-images.githubusercontent.com/23668037/146289127-4ad39b1c-2cff-46aa-adfb-5b02b882e2c3.png">

Reply:
Sure; addressed in f6e5ad4c.